examples/alerts/alerts.md

```diff
@@ -403,7 +403,7 @@ rules:
     )
   for: 5m
   labels:
-    severity: critical
+    severity: warning
```

Comment:
Downgraded to `warning` because  certain amount of failures happen when rolling out receivers,

Reply:
Hm. Still if we are not rolling, we should have it critical, right? Then downgrading might be not great. If we are fine with user to have higher latency as we roll out we should have that in our alerting as well. I guess this means having this alert being critical but less strict. WDYT? (:

Reply:
this should always be warning in my opinion, these are internal requests, the thing that should only ever be critical really for the receive components is the inability to write data, which we do already have :)

Reply:
The thing is. With data-replication one can easily tolerate up to 20% of internal errors, for example, and to the outside, a user won't even notice.

Reply:
Wait. That's not error. That's latency.